### PR TITLE
MC-12823: create custom fields docs

### PR DESCRIPTION
**Pull request by evanluc**
Added docs for creating a custom field

### CODE REVIEW

File: source/includes/administration/_custom_fields.md [hunk 1]
```diff
@@ -47,6 +48,7 @@ Attributes | &nbsp;
 `id`<br/>*UUID* | The custom field's id.
 `organization.id`<br/>*UUID* | The organization id that the custom field belongs to.
 `field`<br/>*UUID* | The field you're defining, this is *not* the UI-facing name. 
+`required`<br/>*boolean* | Whether or not the field is required on the field.
```
Comment (fyrnaga) [inline source/includes/administration/_custom_fields.md:51]: Could mention as optional default is false...

**Review by fyrnaga — APPROVED** [review]
lgtm

File: source/includes/administration/_custom_fields.md [hunk 2]
```diff
@@ -89,6 +92,47 @@ Attributes | &nbsp;
...
+
+Required | &nbsp;
+---------- | -----------
+`field`<br/>*UUID* | The field you're defining, this is *not* the UI-facing name.
+`nameTranslations`<br/>*map* | Map of language short codes to name translations for the field. Translations for all languages are required.
+`type`<br/>*string* | The type of model for the field. Possible values: ORGANIZATION, USER. You will need the `Custom fields: Manage` permission to execute this operation.
+
+Optional | &nbsp;
+---------- | -----------
+`descriptionTranslations`<br/>*map* | Map of language short codes to description translations for the field. If passed translations for all languages are required.
```
Comment (GuillaumeVialle) [inline source/includes/administration/_custom_fields.md]: `If passed, translations` easier to understand with a comma

File: source/includes/administration/_custom_fields.md [hunk 3]
```diff
@@ -89,6 +92,47 @@ Attributes | &nbsp;
...
+`organization.id`<br/>*UUID* | The organization id that the email settings are linked to. It cannot be changed. If not passed will default to the calling user's organization.
```
Comment (GuillaumeVialle) [inline source/includes/administration/_custom_fields.md]: `If not passed, will default` same here

**Review by GuillaumeVialle — APPROVED** [review]
Looks good besides 2 comments

File: source/includes/administration/_custom_fields.md (hunk 3 above)
Comment (evanluc) [inline source/includes/administration/_custom_fields.md]:
```suggestion
`organization.id`<br/>*UUID* | The organization id that the email settings are linked to. It cannot be changed. If not passed, will default to the calling user's organization.
```

File: source/includes/administration/_custom_fields.md (hunk 2 above)
Comment (evanluc) [inline source/includes/administration/_custom_fields.md]:
```suggestion
`descriptionTranslations`<br/>*map* | Map of language short codes to description translations for the field. If passed, translations for all languages are required.
```

**Pull request merged**